loom/script/native/lsNativeDelegate.cpp

```diff
@@ -28,7 +28,7 @@
 #include "loom/script/native/lsNativeDelegate.h"
 
 
-lmDefineLogGroup(gNativeDelegateGroup, "script.native", 1, LoomLogInfo);
+lmDefineLogGroup(gNativeDelegateGroup, "script.ntv", 1, LoomLogInfo);
```

Comment:
Or.. maybe we can just do `ls` instead of `script` and then we can have `ls.native` and `ls.iface`?